lib/tasks/test-sass-compilation.js

```diff
@@ -62,26 +91,24 @@ module.exports = function (cfg) {
 				{
 					title: 'Tests starting...',
 					task: async (context, task) => {
-						const brands = await files.getModuleBrands();
-						task.title = `Testing compilation with silent mode on`;
+						let brands = await files.getModuleBrands();
+						brands = brands.length > 0 ? brands : [
```

Comment:
Let's leave updating brands to meet new spec changes, e.g. 'universal', as a separate PR